#!/usr/bin/env python
PACKAGE = 'leg_detector'
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator() 

gen.add('connection_threshold',     double_t,   0, "[m]", 0.06, 0, 0.25)
gen.add('leg_pair_separation',      double_t,   0, '[m]', 0.5, 0, 1)
gen.add('max_track_jump',           double_t,   0, '[m]', 0.5, 0, 5)
gen.add('no_observation_timeout',   double_t,   0, 'Timeout tolerance for no observations [s]', 4, 0, 5)

gen.add('kalman_p',                 double_t,   0, '',    4, 0, 10)
gen.add('kalman_q',                 double_t,   0, '', .002, 0, 10)
gen.add('kalman_r',                 double_t,   0, '',   10, 0, 20)

exit(gen.generate(PACKAGE, 'leg_detector', 'LegDetector'))
 